languages/x86-64-assembly/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,45 @@
+In x86-64 assembly there are 16 general purpose 64-bit registers, `rax`, `rbx`,
```

Comment:
The contents in this document are great! Love the usage of the ascii art, that will really help. My only comment is that some of the information from the `introduction.md` document seems to be missing here (e.g. the `global` directive). I noticed that [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd) don't make this very clear, so I've just [submitted a PR](https://github.com/exercism/v3/pull/1845) to fix this.

The way I usually go about the `after.md` document, is to start with just copying the `introduction.md` document's contents. Then I'll expand on the information where useful, which could mean more examples or additional information or something else (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd) for a list of possible additions).